test/web/controllers/employee_controller_test.exs

```diff
@@ -85,6 +86,58 @@ defmodule PRM.Web.EmployeeControllerTest do
     assert 2 == length(json_response(conn, 200)["data"])
   end
 
+  test "search employee by tax_id", %{conn: conn} do
+    doctor = "DOCTOR"
+    |> employee()
+    |> Repo.preload(:party)
+    accountant = "accountant"
+    |> employee()
+    |> Repo.preload(:party)
```

Comment:
you can write in one line:
`"accountant" |> employee() |> Repo.preload(:party)`